### ISSUE TITLE
Game rendering produces incorrect colour pallet

### ISSUE THREAD

**Issue by DarthPJB** [nixinator/nil#2]
This needs addressing, as currently a CGA or similar pallet is being used for rendering.

**Comment by nixinator** [comment]
yeah, this might just be the age of the code, or some thing broken with the code, or something with how the old sdl code is setting the pallet.....
